tests/unit/store/mutations.spec.ts

```diff
@@ -133,6 +138,30 @@ describe( 'mutations', () => {
 
 	} );
 
+	it( 'setOmitLabels', () => {
+		const state: RootState = {
+			conditionRows: [
+				{
+					valueData: { value: 'potato', valueError: null },
+					propertyData: { id: 'P666', label: 'Day of the beast', datatype: 'string', propertyError: null },
+					propertyValueRelationData: { value: PropertyValueRelation.Regardless },
+					conditionId: '3',
+				},
+			],
+			limit: 0,
+			useLimit: false,
+			omitLabels: true,
+			errors: [],
+		};
+
+		const omitLabels = true;
+
+		mutations.setOmitLabels( state, omitLabels );
```

Comment:
This mutation is changing `omitLabels` from `true` to `true`. I think it would test more if one of them were false :)